src/Core/src/Platform/Windows/MauiWebView.cs

```diff
@@ -143,12 +153,15 @@ void SetupPlatformEvents()
 			NavigationStarting += (sender, args) =>
 			{
 				// Auto map local virtual app dir host, e.g. if navigating back to local site from a link to an external site
-				if (args?.Uri?.ToLowerInvariant().StartsWith(LocalScheme.TrimEnd('/').ToLowerInvariant()) == true)
+				if (IsUriWithLocalScheme(args?.Uri) ||
+					(IsDataUri(args?.Uri) && _navigatingToHtmlWithBaseUrl))
```

Comment:
Do you have a suggestion on how to solve it? I couldn't find a way because the `NavigationStarting` event is not tied to the `LoadHtml` call.

Reply:
I re-checked the code, and there's no way to pass the flag from `LoadHtml()` to the `NavigationStarting` event. There's the possibility, though, that the data URI could be cheked if it contains the base tag and if the base tag contains the `LocalScheme`. Additionally `WebView2` base64 encodes the whole HTML text, so this must be decoded first. You can imagine that the code for this will get a bit longer. Should I do that?

Reply:
This is corrected in the commit added to this PR today.